JavaFiddle-Web/src/main/java/ru/javafiddle/web/services/GroupService.java

```diff
@@ -32,86 +38,93 @@
     @EJB
     ProjectBean projectBean;
 
+    @EJB
+    UserBean userBean;
+
+    @EJB
+    AccessBean accessBean;
+
     @POST
     @Consumes(MediaType.APPLICATION_JSON)
     public Response addNewGroupMember(GroupMember newGroupMember, @PathParam("projectHash") String projectHash) {
 
-        try{
+        Project project = projectBean.getProjectByProjectHash(projectHash);
+        if (project == null) {
+            return Response.status(Response.Status.NOT_FOUND).build();
+        }
+
+        Group group = project.getGroup();
 
-            int groupId = projectBean.getGroupId(projectHash);
-            //!TODO let's postpone it for later untill signature in GroupBean methods will be established
-//            groupsBean.addMember(groupId,
-//                    newGroupMember.getUserNickName(),
-//                    newGroupMember.getAccessRights());
+        User user = userBean.getUser(newGroupMember.getUserNickName());
+        if (user == null) {
+            return Response.status(Response.Status.NOT_FOUND).build();
+        }
 
-            return Response.ok().build();
+        Access access = accessBean.getAccess(newGroupMember.getAccessRights());
+        if (access == null) {
+            return Response.status(Response.Status.NOT_FOUND).build();
+        }
 
-        } catch (NotFoundException e) {
-            return Response.status(Response.Status.BAD_REQUEST).build();
+        try {
+            groupBean.addMember(group, user, access);
```

Comment:
@alexander1leonov 
I've created an issue for this purpose with assignee for myself.